via/models/transcript.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.orm import Mapped, mapped_column
+
+from via.db import Base
+
+
+class Transcript(Base):
+    __tablename__ = "transcript"
+
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True, init=False)
+    video_id: Mapped[str] = mapped_column(unique=True)
+    transcript: Mapped[list] = mapped_column(JSONB, repr=False)
```

Comment:
We should probably add `created` and `updated` columns to this.

Reply:
> We could grab the mixin for that.

I wonder if SQLAlchemy 2 has a different way but yeah probably.

> Looking ahead, do we want to namespace the transcripts in anyway to account for the possibility of non-Youtube videos which might clash?

Yeah, maybe we should add a `service` column or something whose value would be `"youtube"`, and the unique constraint would be on `(service, video_id)`.

> We could have a Python managed enum for this (which we also have a mixin for I think?). If we are storing huge transcripts saving a few bytes on the type is probably not worth a properly normalized table.

Again I wonder what the SQLAlchemy 2 way might be here